openslides_backend/action/actions/meeting_user/create.py

```diff
@@ -14,6 +16,21 @@ class MeetingUserCreate(CreateAction):
     model = MeetingUser()
     schema = DefaultSchema(MeetingUser()).get_create_schema(
         required_properties=["user_id", "meeting_id"],
-        optional_properties=["comment"],
+        optional_properties=[
+            "comment",
+            "number",
+            "structure_level",
+            "about_me",
+            "vote_weight",
+        ],
     )
     permission = Permissions.User.CAN_MANAGE
+
+    def check_permissions(self, instance: Dict[str, Any]) -> None:
+        if "about_me" in instance and not any(
+            field in instance
+            for field in ("comment", "number", "structure_level", "vote_weight")
+        ):
```

Comment:
Better to ask if there is another field in instance. Your way you have to remember to change the code on adding one more field.

Reply:
There is another field in the instance: user_id and meeting_id. I added `len(instance) == 3` here.